src/intl/bg/page-get-eth.json

```diff
@@ -1,18 +1,18 @@
 {
   "page-get-eth-cex": " Централизирани борси",
   "page-get-eth-cex-desc": "Борсите са бизнеси, които ви позволяват да купувате крипто, използвайки традиционни валути. Те имат контрол върху всички ETH, които купувате, докато не ги изпратите в портфейл под ваш контрол.",
-  "page-get-eth-checkout-dapps-btn": "Вижте дап-овете",
+  "page-get-eth-checkout-dapps-btn": "Вижте dapps",
   "page-get-eth-community-safety": "Публикации на общността за сигурността",
-  "page-get-eth-description": "Етереум и ETH не са под контрола на никое правителство или фирма – те са децентрализирани. Това прави ETH отворен за ползване от всеки.",
+  "page-get-eth-description": "Ethereum и ETH не са под контрола на никое правителство или фирма – те са децентрализирани. Това прави ETH отворен за ползване от всеки.",
   "page-get-eth-dex": "Децентрализирани борси (DEXs)",
   "page-get-eth-dex-desc": "Ако искате повече контрол, тогава купувайте ETH от точка-до-точка. При DEX може да търгувате, без да разрешавате на дадена централизирана компания да контролира сумите ви.",
   "page-get-eth-dexs": "Децентрализирани борси (DEXs)",
   "page-get-eth-dexs-desc": "Децентрализираните борси са отворени пазари за ETH и други токени. Те директно свързват купувачите с продавачите.",
   "page-get-eth-dexs-desc-2": "Вместо да използват доверена трета страна да следи за сигурността на сумите в трансакцията, те използват код. ETH на продавача може да бъде прехвърлен само когато плащането е гарантирано. Този вид код е известен като умен договор.",
   "page-get-eth-dexs-desc-3": "Това означава, че има по-малко географски ограничения отколкото при централизираните алтернативи. Ако някой продава това, което искате, и приема начина ви на плащане, който предлагате, сделката е готова. DEX ви позволява да купите ETH с други токени, чрез PayPal или даже доставени лично в брой.",
-  "page-get-eth-do-not-copy": "ПРИМЕР: НЕ КОПИРАЙТЕ",
+  "page-get-eth-do-not-copy": "Пример: Не копирайте",
```

Comment:
I think we want this uppercase?

<img width="725" alt="Screenshot 2022-01-06 at 15 05 56" src="https://user-images.githubusercontent.com/62268199/148403732-86bf09b2-cb0c-4e53-9459-a6a86e5f42f5.png">

```suggestion
  "page-get-eth-do-not-copy": "ПРИМЕР: НЕ КОПИРАЙТЕ",
```